lib/workers/repository/updates/branch-name.ts

```diff
@@ -64,6 +64,9 @@ export function generateBranchName(update: RenovateConfig): void {
         update.groupSlug = `major-${update.groupSlug}`;
       }
     }
+    if (update.updateType === 'minor' && update.separateMultipleMinor) {
+      update.groupSlug = `minor-${update.newMajor}.${update.newMinor}-${update.groupSlug}`;
```

Comment:
I think the type for `update` is wrong as function parameter. it should be a type which has all the missing properties